tools/metrics/gauges_uaa_test.go

```diff
@@ -82,4 +84,161 @@ var _ = Describe("UAA", func() {
 			))
 		})
 	})
+
+	Context("Active Users", func() {
+		It("Should select active users", func() {
+			activeUsers := []uaaclient.User{{
+				Origin:        "uaa",
+				LastLogonTime: int(time.Now().Add(-1*24*time.Hour).Unix() * 1000),
+			}}
+
+			Expect(UAAActiveUsers(activeUsers)).To(HaveLen(1))
+		})
+
+		It("Should reject inactive users", func() {
+			inactiveUsers := []uaaclient.User{{
+				Origin:        "uaa",
+				LastLogonTime: int(time.Now().Add(-1*24*time.Hour*31).Unix() * 1000),
+			}}
+
+			Expect(UAAActiveUsers(inactiveUsers)).To(HaveLen(0))
+		})
+
+		It("Should select active and reject inactive users", func() {
+			users := []uaaclient.User{{
+				Origin:        "google",
+				LastLogonTime: int(time.Now().Add(-1*24*time.Hour).Unix() * 1000),
```

Comment:
Maybe extract `-1*24*time.Hour` expression to a named constant, to make the test easier to comprehend?

`int(time.Now.Add(MINUS_ONE_DAY).Unix() * 1000)` or even `int(ONE_DAY_AGO)`?